Source/AliveLibAE/Input.cpp

```diff
@@ -1362,6 +1362,8 @@ EXPORT int Input_Convert_KeyboardGamePadInput_To_Internal_Format_492150()
                 {
                     buttons = pButtons;
 
+                    unsigned int shoulderButtonsPressedCount = 0; // Added for chanting with any shoulder button combo
```

Comment:
I would add a comment marking this as an OG change so it can easily be looked/up spotted when fixing any reversing bugs

Reply:
Noted. Should I say "OG change" for each comment, just here, or maybe on the if(shoulderButtonsPressedCount > 1) line, where the change mostly occurs?

Reply:
if you're gonna keep both I'd just have an IsChanting() and IsChantingAnyShoulderButton() method or something and put them in that, then the call it only needs the comment and not every line